docs/modules/logstash.md

```diff
@@ -1,8 +1,11 @@
 # Logstash
 
 ### Purpose
+[Logstash](https://www.elastic.co/products/logstash) is an open-source tool for collecting, parsing, and storing logs for future use. Logstash will run alongside Kibana and Elasticsearch. 
 
 ### Dependencies
+Java 8 is required, also Elasticsearch will need to be downloaded beforehand since Logstash is a plugin for Elasticsearch. Logstash can be downloaded as a Tarball, ZIP, DEB, and RPM through their [website](https://www.elastic.co/downloads/logstash). Please refer to their download instructions to download from package repositories on their [website](https://www.elastic.co/guide/en/logstash/current/installing-logstash.html#package-repositories). Please refer to the [docs](https://www.elastic.co/guide/en/logstash/current/index.html) for any additional questions.
```

Comment:
I just want to call out this section as being real good, the overall theme is totally within the scope of what we're trying to document. The one thing I would change is researching whether Elasticsearch is actually a dependency or if Logstash can operate independently. You can check out the Logstash docs on "output plugins" for more info.